### PR TITLE
display who reviewed prs

### PR DESCRIPTION
**Pull request by FaithMo**
Related issues: [display who has done PR reviews recently, similar to how we display who has done recent competence reviews](https://www.notion.so/display-who-has-done-PR-reviews-recently-similar-to-how-we-display-who-has-done-recent-competence-r-a9d130027d0244558c07214f73f18469)

## Description:

What are you up to? Fill us in :)

## I solemnly swear that:

- [x] My code follows the style guidelines of this project
- [x] I have merged the develop branch into my branch and fixed any merge conflicts
- [x] I have performed a self-review of my own code
- [ ] I have commented my code in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: frontend/src/components/pages/GlobalCodeReviewDashboard/BaseQueryEntry.jsx [hunk 1]
```diff
@@ -81,12 +83,8 @@ export default function BaseReviewQueueEntry({
           <ReviewersTable
             reviewerUsers={project.reviewerUsers}
             reviewerUserEmails={project.reviewerUserEmails}
-            usersThatReviewedSinceLastReviewRequest={
-              project.usersThatReviewedSinceLastReviewRequest
-            }
-            usersThatReviewedSinceLastReviewRequestEmails={
-              project.usersThatReviewedSinceLastReviewRequestEmails
-            }
+            usersThatReviewedSinceLastReviewRequest={reviewers}
```
Comment (Ngoakor12) [inline frontend/src/components/pages/GlobalCodeReviewDashboard/BaseQueryEntry.jsx]: - Should the values passed to this not contain id's and not emails? I'm saying this looking at the first two props for ReviewersTable

- Would passing in both the reviewer ids and emails as props not make it easier to deal with? maybe you might not need the complex `filterUsers` function and more manipulation in the `ReviewersTable.jsx`?

**Review by Ngoakor12 — CHANGES_REQUESTED** [review]
- try and resolve this warning generated by your code
image

- Regarding the main task at hand, I might be wrong but I think closely following what was done on the competence reviwers section may mean you don't need to touch the `ReviewersTable.jsx` file an the utils files aswell, left a comment  about this below

File: frontend/src/components/pages/GlobalCodeReviewDashboard/BaseQueryEntry.jsx (hunk 1 above)
Comment (FaithMo) [reply frontend/src/components/pages/GlobalCodeReviewDashboard/BaseQueryEntry.jsx:86]: `ReviewersTable.jsx` is currently only catered for the competence reviews, and the dataset has two different reviewers, the PR reviews and the competence review, so I needed a way to allow the table to take in two different reviewers

**Review by Ngoakor12 — COMMENTED** [review]
Should the `package-lock.json` be changed if you're not installing anything new?

Otherwise it looks good to me

**Review by Ngoakor12 — APPROVED** [review]
looks good to me

**Review by Vuyisanani — APPROVED** [review]

**Pull request closed without merging**